ui/pages/settings/networks-tab/custom-content-search/custom-content-search.test.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```

Comment:
I just noticed that this test uses Enzyme. Despite the fact that a lot of our tests use Enzyme, we've actually deprecated it in favor of React Testing Library. Would you mind rewriting this test to use that instead? You can see an example of a test written using React Testing Library here: https://github.com/MetaMask/metamask-extension/blob/d15de941e3572af989be500ef67b0418129973aa/ui/components/app/edit-gas-display/edit-gas-display.test.js. There are some different paradigms when using RTL, one of which is that instead of searching for elements by id or class, you just search for them by text. Also, there is no need to render your component with a context that provides a stub for I18nContext, as that happens automatically, so you don't have to worry about i18n — it "just works". Let me know if you need help with this.

Reply:
@mcmire Done [here](https://github.com/MetaMask/metamask-extension/pull/13560/commits/c48e1890932565addc62ccd37f3dfad1a324c708)